src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -51,6 +54,8 @@ public FeedbackResponseAttributes(String feedbackSessionName,
         this.recipientEmail = recipientEmail;
         this.recipientSection = recipientSection;
         this.responseMetaData = responseMetaData;
+        this.createdAt = new Date();
+        this.updatedAt = new Date();
```

Comment:
`*Attributes` are not required to have it. Only when it is constructed from its corresponding entity then the `createdAt` and `updatedAt` fields need to be preserved. So here they are unnecessary.